internal/state/registry.go

```diff
@@ -29,6 +30,18 @@ func sFnRegistryConfiguration(ctx context.Context, r *reconciler, s *systemState
 		setK3dRegistry(s)
 	}
 
+	if condition, ok := s.instance.GetCondition(v1alpha1.ConditionTypeConfigured); ok {
+		if condition.Status == metav1.ConditionFalse {
+			s.setState(v1alpha1.StateReady)
```

Comment:
I'm not sure if setting the state to Ready is ok in this case. The ready state does mean that ALL conditions are in the True state (resource is configured and installed)